docs/how-to/configure-rbac.mdx

```diff
@@ -1,29 +1,79 @@
+These permissions are expressed through scopes, which are strings that represent the set of permissions.
+
+For our example above, we would create a resource server called Shoes and the scopes that are available could be `shoes:read`, `shoes:buy`, `clearance:buy`, `new-arrivals:read`, `new-arrivals:buy`, etc.
+
+## Roles
+
+Roles map to a set of allowed scopes. You then assign identities to these roles.
+
+For our example above, we would create the following roles and their scopes:
+
+- Free Member: `shoes:read`, `shoes:buy`, `clearance:buy`
+- VIP Member: `shoes:read`, `shoes:buy`, `clearance:buy`, `new-arrivals:read`, `new-arrivals:buy`
+
+Roles come into play in two spots: (1) Token Minting and (2) Token Introspection.
+
+## Token Minting
+
+Whenever an access token is minted on behalf of an identity, the scopes on that token is the intersection of the three following scope sets:
+
+1. Scopes on the `/authorize` request (during the OAuth2 / OIDC flow).
+1. Scopes supported by the Application the identity is authenticating into.
+1. Scopes tied to the identity's roles.
+
+For our example above, a free member authenticates into the website would have an access token with the scopes `shoes:read`, `shoes:buy`, `clearance:buy`, and a VIP
+member's access token would contain the scopes `shoes:read`, `shoes:buy`, `clearance:buy`, `new-arrivals:read`, `new-arrivals:buy`.
+
+## Token Introspection
+
+Whenever a [token introspection](<link to token introspection>) is called for an access token with an identity as the principal, the scopes returned in the introspect
+response will contain **only a subset of the scopes for which the identity has the role(s) for**.
+
+For the most part, the scopes in the token introspection response will match the scopes on the token itself unless permissions change. But should a user's roles change,
+the introspect response will reflect those changes.
+
+For our example above, say a VIP member logs into the website and halfway through their session, they lose their VIP status. Now when the (now free) member attempts to view
+the new arrivals, the introspect response should NOT return the `new-arrivals:read` scope on the response and the user should be denied access to that page.
+
+## Putting it all together
+
+After creating the aforementioned resource server and roles above, we would create an application with the authorization code flow and integrate that with your application.
+Now, whenever a user logs in and attempts to access the various parts of your website, RBAC will ensure that the logged in user can only access the resources that are allowed
+to them by their role.
+
+RBAC is an authorization tool that can be used to validate access permissions on the end user, and is just one of the tools that Beyond Identity can help you accomplish your
```

Comment:
Is there a logical next place to link a user for more tools to accomplish identity and access management goals?

Reply:
I don't think so, this is it!